components/sceneutil/unrefqueue.cpp

```diff
@@ -1,28 +1,18 @@
 #include "unrefqueue.hpp"
 
-#include <deque>
-
 //#include <osg/Timer>
 
 //#include <components/debug/debuglog.hpp>
-#include <components/sceneutil/workqueue.hpp>
 
 namespace SceneUtil
 {
-
-    class UnrefWorkItem : public SceneUtil::WorkItem
+    void UnrefWorkItem::doWork()
     {
-    public:
-        std::deque<osg::ref_ptr<const osg::Referenced> > mObjects;
-
-        virtual void doWork()
-        {
-            //osg::Timer timer;
-            //size_t objcount = mObjects.size();
-            mObjects.clear();
-            //Log(Debug::Verbose) << "cleared " << objcount << " objects in " << timer.time_m();
-        }
-    };
+        //osg::Timer timer;
+        //size_t objcount = mObjects.size();
+        mObjects.clear();
+        //Log(Debug::Verbose) << "cleared " << objcount << " objects in " << timer.time_m();
```

Comment:
Huh. You probably can remove these debugging leftovers.

Reply:
Is this a blocking change?

It being my first PR, I was going for as low-impact as possible...

Reply:
We generally do not encourage pure code cleanup changes, because they increase the chance of merge conflicts. However if you are already touching lines of code that need cleanup, then you should do the cleanup.

There shouldn't be any code that is commented out. That's what we got git for. Usually we catch that in the code review, but sometimes it can slip through.

Just add a new commit to this PR that removes the comments.

Reply:
Got it - I committed the change